pkg/volume/vsphere_volume/vsphere_volume_util.go

```diff
@@ -122,19 +129,57 @@ func verifyDevicePath(diskUUID string) string {
 	return ""
 }
 
+// scans storage devices and adds path in dev/disk/by-id
 func probeAttachedVolume() error {
-	executor := exec.New()
-	args := []string{"trigger"}
-	cmd := executor.Command("/usr/bin/udevadm", args...)
-	_, err := cmd.CombinedOutput()
+	var (
+		scsiPath   = "/sys/class/scsi_host/"
+		driverType = "mpt"
```

Comment:
I am not sure if this works for VMs that use scsi controller other than LSI. Have you tested that? If it is not an easy fix, please make a note somewhere.